staging/src/k8s.io/apiserver/pkg/server/healthz/healthz.go

```diff
@@ -187,14 +187,20 @@ func adaptCheckToHandler(c func(r *http.Request) error) http.HandlerFunc {
 
 // checkerNames returns the names of the checks in the same order as passed in.
 func checkerNames(checks ...HealthzChecker) []string {
```

Comment:
I don't think we use `checkerNames` now.  Maybe just rename the function to `formatCheckerNames` but keep the existing code, and change the comment to say that we are formatting them for printing/debugging...